text/2021-10-08-monitoring-metrics-about-chaos-mesh.md

```diff
@@ -0,0 +1,241 @@
+| chaos_controller_manager_chaos_schedule_count         | currrent count of Schedule                             | GaugeVec     | namespace                   | /                            |
+| chaos_controller_manager_chaos_workflow_count         | currrent count of Workflow                             | GuageVec     | namespace                   | /                            |
+| chaos_controller_manager_emitted_event_count          | count of the emitted event by chaos-controller-manager | GaugeVec     | type, reason                | /                            |
+| chaos_controller_manager_grpc_client_handling_seconds | common metrics of grpc-client                          | HistogramVec | code, method, service, type | DefBuckets                   |
+| chaos_controller_manager_webhook_duration_seconds     | metrics for kubernetes webhook - duration              | HistogramVec | type, allowed               | WebhookDurationBuckets       |
+| chaos_controller_manager_webhook_request_total        | metrics for kubernetes webhook - request count         | CountVec     | type, allowed               | /                            |
+| chaos_daemon_grpc_server_handling_seconds             | common metrics of grpc-server                          | /            | /                           | ChaosDaemonGrpcServerBuckets |
+| chaos_daemon_bpm_controlled_process_total             | total of processes controlled by `bpm`                 | Count        | /                           | /                            |
+| chaos_daemon_bpm_controlled_process_count             | current count of processes controlled by `bpm`         | Guage        | /                           | /                            |
+| chaos_daemon_iptables_chain_count                     | current count of iptables chains                       | GaugeVec     | container_id                | /                            |
+| chaos_daemon_iptables_rule_count                      | current count of iptables rules                        | GaugeVec     | container_id, table, chain  | /                            |
+| chaos_daemon_iptables_packet_count                    | current count of all packet                            | GaugeVec     | container_id, table, chain  | /                            |
+| chaos_daemon_iptables_packet_bytes                    | current values of all packet byte                      | GaugeVec     | container_id, table, chain  | /                            |
+| chaos_daemon_ipset_member_count                       | current count of members in an ipset                   | GuageVec     | container_id, name          | /                            |
+| chaos_daemon_tc_count                                 | current count of tc rules                              | GuageVec     | container_id                | /                            |
```

Comment:
Not sure if using container id is good. Can we use pod name and container name as label at the same time?